src/details/actions.js

```diff
@@ -1,11 +1,21 @@
 import moment from 'moment';
 import numeral from 'numeral';
+import hash from 'string-hash';
+import ls from 'local-storage';
 
 import * as d from './actionTypes';
 
 // Mock data to avoid hammering the API
 // import json from './sample/usageRecords.json';
```

Comment:
Should this be removed as it is commented out?

Reply:
Sure, l guess this should be transferred to unit test, I've created #10 to take care of it.